### PR TITLE
ROX-17751: Add GKE target for sensor-integration tests

### PR DESCRIPTION
**Pull request by vikin91**
Requires https://github.com/stackrox/stackrox/pull/6480 for the rehearsals to pass

### CODE REVIEW

**Comment by vikin91** [comment]
Requires https://github.com/stackrox/stackrox/pull/6416 for the rehearsals to pass

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-x.y-gke-sensor-integration-tests

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml [hunk 1]
```diff
@@ -61,6 +61,11 @@ tests:
   steps:
     test:
     - ref: stackrox-e2e
+- as: gke-sensor-integration-tests
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml:64]: You will need to add a `merge-` a `postsubmit: true` version of this.

File: ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml [hunk 2]
```diff
@@ -463,6 +463,63 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )gke-scale-tests,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (gavin-stackrox) [inline ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml:467]:
```suggestion
    always_run: false
```
And use `skip_if_only_changed: ^ui/` https://docs.prow.k8s.io/docs/jobs/#triggering-jobs-based-on-changes

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies.yaml [hunk 3]
```diff
@@ -93,6 +93,11 @@ tests:
   steps:
     test:
     - ref: stackrox-e2e
+- as: gke-sensor-integration-tests
+  optional: true
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies.yaml:97]: All of nightlies are postsubmit jobs.
```suggestion
  postsubmit: true
```

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies.yaml (hunk 3 above)
Comment (vikin91) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies.yaml]: Added that in [fec3275](https://github.com/openshift/release/pull/40290/commits/fec32759fea1fdc1840a8e55a250542dc96f125c)

File: ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml (hunk 2 above)
Comment (vikin91) [reply ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml]: Added that in [fec3275](https://github.com/openshift/release/pull/40290/commits/fec32759fea1fdc1840a8e55a250542dc96f125c)

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml (hunk 1 above)
Comment (vikin91) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml:64]: Added that in [fec3275](https://github.com/openshift/release/pull/40290/commits/fec32759fea1fdc1840a8e55a250542dc96f125c)

**Review by vjwilson — APPROVED** [review]

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-x.y-gke-sensor-integration-tests

**Pull request merged**